etna/models/nn/deepstate/deepstate.py

```diff
@@ -180,8 +212,16 @@ def forward(self, x: DeepStateBatch, *args, **kwargs):  # type: ignore
 
     def make_samples(self, df: pd.DataFrame, encoder_length: int, decoder_length: int) -> Iterator[dict]:
         """Make samples from segment DataFrame."""
-        values_real = df.drop(columns=["target", "segment", "timestamp"]).values
+        values_real = df.drop(columns=["target", "segment", "timestamp"] + list(self.embedding_sizes.keys())).values
```

Comment:
I think we should rewrite this place in other models because it is more clear and there is no need to find intersection between column like we did

Reply:
Ok, let's do this.